docs/libs/ml/index.md

```diff
@@ -30,7 +122,7 @@ under the License.
 </dependency>
 {% endhighlight %}
 
-## Algorithms
+## Algorithm Documentation
```

Comment:
I think, we should move the algorithms a little bit more prominently to the top of the page.